src/main.cpp

```diff
@@ -149,6 +149,13 @@ namespace
 	{
 		try {
 			auto* input = boost::any_cast<modAVUMetadataInp_t*>(*std::next(std::begin(_rule_arguments), 2));
+
+			if (strcmp(input->arg0, "rmw") == 0) {
+				log_re::error({{"log_message", "rmw is deprecated and unsupported by metadata_guard"},
```

Comment:
I think that this message should communicate that `rmw` is not just unsupported when metadata guard is configured, but expressly disallowed. I guess I'm viewing "unsupported" in light of how REPs treat unsupported/unrecognized rules: pass the buck to the next configured REP by returning `SYS_NOT_SUPPORTED`. Thoughts?

Reply:
I think this is... okay?   We're returning `SYS_NOT_SUPPORTED` below - and this will only be seen by someone who's trying to use a thing that **used** to exist...

Reply:
Oh, I see what you're saying... the message is mentioning metadata_guard and that may not mean anything to the user/client.

Hmmm.

Once rmw is gone gone, would it even make it to this code?   Would the iRODS API itself already have said no?

Reply:
I'm saying that metadata guard doesn't *allow* usage of `rmw`, and saying that it doesn't *support* its usage it's slightly inaccurate (IMO) because it actively denies its usage. In my mind, "not supported" means we are not even going to guard against this action because we don't handle that. We do in fact handle it and return an error.

Hopefully that makes sense. It's not a big deal since it will go away, like you said. Just a semantics thing, perhaps.

Reply:
oh... i get it.   so you're proposing a stronger sentence to be returned..  do you have a suggested new sentence?

Reply:
Maybe something like this?

"rmw is deprecated and its usage is disallowed by metadata_guard."

It's really just the word "supported" vs. "allowed" that messes with my brain.

Reply:
disallowed is fine to me

Reply:
That wording sounds good to me.